apps/crn-server/test/fixtures/research-output.fixtures.ts

```diff
@@ -254,6 +254,20 @@ export const getResearchOutputResponse = (): ResearchOutputResponse => {
   };
 };
 
+export const getRelatedResearchOutputResponse = (
```

Comment:
Yes, I can explain.

In all the tests that I touched on within this PR, `getResearchOutputResponse` is used to build a RO. We use this RO as a mock and we use it to assert that _Algolia was called with this object_.

Since we add a new piece of code that says: "When an RO is created/updated, **re-index all the related ROs**", we need to have as mock data the related ROs so that we can use them.

What this `getRelatedResearchOutputResponse` does is that it takes an RO and just overrides the title, documentType, and so on, just to make sure we have built the related ROs as mock data and assert _that Algolia saves them too_.

<img width="591" alt="Screenshot 2023-05-18 at 11 11 20" src="https://github.com/yldio/asap-hub/assets/14296353/27bc2c50-f5f2-4f9a-a838-93cb1341097a">

All in all, is just a faster way to build the related research outputs that exist in `getResearchOutputResponse` mock that is used in our tests.

Reply:
if its a faster way to get related research then the types are incorrect - the type for related research is not `ResearchOutputResponse` but:
```typescript
Pick<
      ResearchOutputDataObject,
      'id' | 'title' | 'type' | 'documentType' | 'teams'
    > & { isOwnRelatedResearchLink?: boolean }
```

Reply:
Updated accordingly